docs/tutorials/pwm_control_flight_controller.rst

```diff
@@ -306,7 +332,7 @@ section of Mission Planner. Search for the parameters listed below and make sure
 * **SERVO_DSHOT_ESC = 0**
   
   * This controls what type of additonal ESC commands will be sent from the flight controller. These additional commands include things like beeping, LED control, and other potentially useful but non-essential
-    extra commands that can be sent over DSHOT. But this does not need to be enabled in order to send the motor throttle commands over DSHOT, so we are leaving it disabled for simplicity. See the ArduCopter documentation for more details
+    extra commands that can be sent over DSHOT. But this does not need to be enabled in order to send the module throttle commands over DSHOT, so we are leaving it disabled for simplicity. See the ArduCopter documentation for more details
```

Comment:
the standard PWM bullets have periods, but these ones don't. I'd say pick one and keep it consistent.